FirebaseAuth/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# Unreleased
```

Comment:
How about 11.5 for CocoaPods/SPM and 11.6 for zip/Carthage?

cc: @andrewheard

Reply:
That sounds reasonable to me.

Reply:
https://github.com/firebase/firebase-ios-sdk/pull/14076 to address this!